internal/app/wwctl/container/exec/child/main.go

```diff
@@ -21,6 +21,8 @@ import (
 	"github.com/warewulf/warewulf/internal/pkg/wwlog"
 )
 
+const exitEval = `$(VALU="$?" ; if [ $VALU == 0 ]; then echo write; else echo discard; fi)`
```

Comment:
"write" and "discard" are a bit misleading here, aren't they? Changes stay resident in the chroot; they just aren't built into a new image.

Maybe "build" and "skip"? Then again, if I understand correctly, I think this change is out-of-scope for this PR, and should be split into a different PR, probably reserved for v4.6.0.

Reply:
Or have I misunderstood? Is it actually the case that changes are discarded on nonzero exit now?

_goes to test_

Reply:
No, the files stay resident.

```
[root@wwctl2 ~]# wwctl container shell rocky-8
[rocky-8] Warewulf> touch /testfile
[rocky-8] Warewulf> exit -1
exit
Command exited non-zero, not rebuilding/updating VNFS image

[root@wwctl2 ~]# stat /var/lib/warewulf/chroots/rocky-8/rootfs/testfile
  File: /var/lib/warewulf/chroots/rocky-8/rootfs/testfile
  Size: 0         	Blocks: 0          IO Block: 4096   regular empty file
Device: fd00h/64768d	Inode: 51694332    Links: 1
Access: (0644/-rw-r--r--)  Uid: (    0/    root)   Gid: (    0/    root)
Context: unconfined_u:object_r:var_lib_t:s0
Access: 2024-04-20 01:41:46.232917500 -0400
Modify: 2024-04-20 01:41:46.232917500 -0400
Change: 2024-04-20 01:41:46.232917500 -0400
 Birth: 2024-04-20 01:41:46.232917500 -0400
```